src/app/tabs/settings-tab.js

```diff
@@ -10,9 +11,9 @@ var helper = require('../../lib/helper')
 var modal = require('../ui/modal-dialog-custom')
 var tooltip = require('../ui/tooltip')
 
-var css = require('./styles/settings-tab-styles')
+var styles = require('../../ui/styles-guide/theme-chooser').chooser()
 
-function SettingsTab (appAPI = {}, appEvents = {}, opts = {}) {
+module.exports = function SettingsTab (appAPI = {}, appEvents = {}, opts = {}) {
```

Comment:
should it be ES6 `class`

Reply:
yes, agree.
I am actually right now working on the settings tab.
The settings tab PR i still had could not easily be translated, because it had so many conflicts, that i decided to do this one again, because it seems faster than resolving all those complicated conflicts.